css/main.css

```diff
@@ -1,3 +1,271 @@
+}
+
+body {
+  background-color: white;
+  position: relative;
+}
+
+body:after {
+  top: 0;
+  left: 0;
+  width: 35%;
+  min-width: 500px;
+  height: 1200px;
+  position: absolute;
+  z-index: -1;
+  background-color: #e6dace;
+  content: "";
+}
+
+/* Navigation bar */
+
+.navbar-toggler:focus,
+.navbar-toggler:active,
+.navbar-toggler-icon:focus {
+    outline: none;
+    box-shadow: none;
+}
+
+.navbar.center .navbar-inner .nav {
+  display: inline-block;
+  float: none;
+}
+
+.navbar {
+  background-color: white;
+  color: rgb(107, 10, 136);
+}
+
+.navbar-brand,
+.navbar-brand:hover {
+  color: rgb(107, 10, 136);
+}
+
+.navbar-light .navbar-nav .nav-link {
+  color: #49274a;
+  cursor: pointer;
+  margin-right: 1em;
+  justify-content: space-between;
+}
+
+.navbar-light .navbar-nav .nav-link:hover {
+  padding-left: 10px;
+  padding-right: 10px;
+  font-weight: bolder;
+  color: rgb(107, 10, 136);
+  border-radius: 5px;
+}
+
+.navbar-light .navbar-toggler {
+  background: rgba(58, 60, 61, 0.062);
+  border-color: #49274a;
+}
+
+.card-title {
+  /* font-family: Tahoma; */
+  font-weight: bolder;
+}
+
+/* Main */
+
+.business-card {
+  background-color: rgb(244, 236, 230);
+  width: 375px;
+  height: auto;
+}
+
+.profile-pic {
+  width: 120px;
+  height: auto;
+  border-radius: 50%;
+  margin: 10px;
+}
+
+.icons {
+  width: 100%;
+  padding: 10px;
+  display: flex;
+  justify-content: center;
+}
+
+.icons i {
+  padding-right: 10px;
+  padding-left: 10px;
+}
+
+ul.social-media-list img {
+  padding: 4px;
+  border-radius: 5px;
+  background-color: white;
+  width: 25px;
+  height: 25px;
+}
+
+ul.social-media-list {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  font-size: 2px;
+}
+
+ul.social-media-list li {
+  display: inline-block;
+  margin: 1px;
+}
+
+.line {
+  color: purple;
+}
+
+.resume,
+.portfolio {
+  color: purple;
+  padding: 5px 10px;
+  text-align: center;
+  text-decoration: none;
+  border-color: purple;
+  display: inline-block;
+  justify-self: center;
+  margin: 10px 5px;
+  cursor: pointer;
+  border-radius: 18px;
+}
+
+.resume {
+  background-color: purple;
+  color: white;
+}
+
+.resume:hover {
+  background-color: #fff;
+  color: #49274a;
+  border-color: #49274a;
+}
+
+.portfolio:hover {
+  background-color: #49274a;
+  border-color: #49274a;
+}
+
+/* Footer */
+footer {
+  font-size: small;
+  position: fixed;
+  /* height: auto; */
+  bottom: 0;
+  width: 100%;
+}
+
+/* Contact page */
+
+.portfolio-body {
+  background-color: #e6dace;
+}
+
+input[type=text], #form_email, #form_message, textarea{
+  width: 100%;
+  padding: 12px;
+  border: 2px solid #ccc;
+  border-radius: 7px;
+  box-sizing: border-box;
+  resize: vertical;
+}
+
+label {
+  padding: 12px 12px 12px 0;
+  display: inline-block;
+}
+
+input[type=submit] {
+  background-color: #49274a;
+  color: white;
+  padding: 12px 20px;
+  border: none;
+  border-radius: 7px;
+  cursor: pointer;
+  float: right;
+}
+
+.contact {
+  width: 60%;
```

Comment:
you can use ``col-sm-7`` or ``col-sm-8`` bootstrap class so you don't need to set the width manually in the CSS